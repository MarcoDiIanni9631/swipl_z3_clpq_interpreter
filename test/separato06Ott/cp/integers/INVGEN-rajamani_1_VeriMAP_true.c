# 1 "MAP/SAFE-exbench/INVGEN-rajamani_1.tmp.c"
# 1 "<command-line>"
# 1 "MAP/SAFE-exbench/INVGEN-rajamani_1.tmp.c"
# 19 "MAP/SAFE-exbench/INVGEN-rajamani_1.tmp.c"
int __BLAST_NONDET;

int main(){
  int x=0;
  int y=0;
  int z=0;
  int w=0;

  while ( __BLAST_NONDET ){
    if ( __BLAST_NONDET ) {
      x++; y = y+100;
    } else if ( __BLAST_NONDET ) {
      if( x >= 4)
 { x=x+1; y=y+1;}
    } else if ( y >10*w)
      if (z>=100*x )
      y = -y;
    w=w+1;
    z=z+10;
  }
  if ( x >=4 )
    __VERIFIER_assert( y>2 );
}
