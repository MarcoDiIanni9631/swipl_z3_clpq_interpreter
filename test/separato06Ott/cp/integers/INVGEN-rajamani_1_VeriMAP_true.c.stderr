MAP/SAFE-exbench/INVGEN-rajamani_1.tmp.c:39: Warning: Body of function main falls-through. Adding a return statement
